content/source/docs/cloud/api/run.html.md

```diff
@@ -81,6 +81,7 @@ Properties without a default value are required.
 
 Key path                    | Type   | Default | Description
 ----------------------------|--------|---------|------------
+`data.attributes.auto-apply` | bool | false | Whether to automatically apply changes when a Terraform plan is successful. Defaults to the [Workspace auto-apply](https://www.terraform.io/docs/cloud/workspaces/settings.html#auto-apply-and-manual-apply).
```

Comment:
* I'd make the default value in the column more obvious than just `false`, even with the added description.
* Conventionally things like Workspace aren't capitalized in the docs.

```suggestion
`data.attributes.auto-apply` | bool | (The current Auto Apply option of the workspace) | Whether to automatically apply changes when a Terraform plan is successful. Defaults to the [workspace Auto Apply](https://www.terraform.io/docs/cloud/workspaces/settings.html#auto-apply-and-manual-apply) setting.
```